cli/flags.rs

```diff
@@ -193,6 +195,37 @@ Prettier dependencies on first run.
             .multiple(true)
             .required(true),
         ),
+    ).subcommand(
+      SubCommand::with_name("xeval")
+        .setting(AppSettings::DisableVersion)
+        .about("Eval a script on text segments from stdin")
+        .long_about(
+          "
+Eval a script on lines (or chunks split under delimiter) from stdin.
+
+Read from standard input and eval code on each whitespace-delimited
+string chunks.
+
+-I/--replvar optionally set variable name for input to be used in eval.
+Otherwise '$' will be used as default variable name.
+
+  cat /etc/passwd | deno xeval \"a = $.split(':'); if (a) console.log(a[0])\"
+  git branch | deno xeval -I 'line' \"if (line.startsWith('*')) console.log(line.slice(2))\"
+  cat LICENSE | deno xeval -d ' ' \"if ($ === 'MIT') console.log('MIT licensed')\"
+",
+        ).arg(
+          Arg::with_name("replvar")
+            .long("replvar")
+            .short("I")
+            .help("Set variable name to be used in eval, defaults to $")
+            .takes_value(true),
+        ).arg(
+          Arg::with_name("delim")
+            .long("delim")
+            .short("d")
+            .help("Set delimiter, defaults to newline")
```

Comment:
Is it newline or whitespace? The subcommand help says "whitespace-delimited".

(IMO line sep seems more useful as a default... but okay.)

Reply:
Ooopsy, it should be beeline (though it is funny to see that `xargs` also claims to be `white space delimited` while actually working on new lines also)

This would be fixed along with other changes made to this feature as discussed above